dataset/dictionary/y-events.json5

```
@@ -155,50 +176,58 @@
   {
     en: "Star-Seeker's Sojourn",
     ja: "星探しの旅",
+    zhCN: "寻星之旅",
     pronunciationJa: "ほしさがしのたび",
     notes: "v3.1 期間限定イベント",
     tags: [ "event" ],
   },
   {
     en: "Wind Chaser",
     ja: "五風巡察",
+    zhCN: "循察五风",
     pronunciationJa: "ごふうじゅんさつ",
     notes: "v3.1 期間限定イベント",
     tags: [ "event" ],
   },
   {
     en: "Weinlesefest",
     ja: "ブリーズブリュー祭",
+    zhCN: "归风佳酿节",
```

In the Japanese version, this festival is called ブリーズブリュー祭 in the v3.1 PV, but it is actually called ブリュー祭 in the game. Both are called Weinlesefest in English version.
Is there such a different name in the Chinese version? I will add ブリュー祭 later, so if there is the corresponding word to ブリュー祭 in Chinese, I want to add it.

---

Yes, it is 佳酿节 in Chinese.

---

OK, then let's make like this:

```suggestion
    zhCN: "佳酿节 / 归风佳酿节",
```